packages/cli/src/commands/dataMigrate/up.js

```diff
@@ -1,24 +1,15 @@
 import fs from 'fs'
 import path from 'path'
 
-import babelRequireHook from '@babel/register'
 import Listr from 'listr'
 import VerboseRenderer from 'listr-verbose-renderer'
 import terminalLink from 'terminal-link'
 
+import { registerApiSideBabelHook } from '@redwoodjs/internal'
+
 import { getPaths } from '../../lib'
 import c from '../../lib/colors'
 
-babelRequireHook({
```

Comment:
I moved this into handler, so that the babel register happens only when you run `dm up`. It's also why I'm importing db once and passing it to other functions